### PR TITLE
Improve BinderHub page Accessibility

### PR DESCRIPTION
**Pull request by bl-aire**
Haven accessed [BinderHub page](https://mybinder.org) using WAVE and axe accessibility tools stated in the [microtask](https://github.com/jupyterhub/outreachy/issues/57#issue-1409736912), I have implemented changes based on my solutions proffered  in the issue which are geared towards improving the accessibility of [BinderHub page](https://mybinder.org).

This PR does the following:

1. Add alt text to images via the alt attribute `alt="example text"`.
2. Add the lang attribute to the opening html tag `<html lang="en>"`.
3. Improve contrast between selected text and their background colors.
4. Fix missing first level heading.

More details about the changes made can be found in [issue #57](https://github.com/jupyterhub/outreachy/issues/57#issue-1409736912)

### CODE REVIEW

File: binderhub/static/index.css [hunk 1]
```diff
@@ -236,18 +236,27 @@ h4 {
...
-    color: rgb(208, 102, 129);
+    border-color: rgb(204, 67, 101);
+    color: rgb(204, 67, 101);
 }
 
 .point-blue {
-    border-color: rgb(87, 154, 202);
-    color: rgb(87, 154, 202);
+    border-color: rgb(41, 124, 184);
+    color: rgb(41, 124, 184);
```
Comment (minrk) [inline binderhub/static/index.css:250]: Are you able to share a screenshot of the page with these changes? If not, I can help with that.

**Review by minrk — COMMENTED** [review]
Wonderful, thank you! I've added some comments.

Let me know if you need a hand on the screenshot.

File: binderhub/static/index.css [hunk 2]
```diff
@@ -236,18 +236,27 @@ h4 {
...
+}
+
+/*reduce font size of h1 and h2 so the initial design when h3 and h4 tags were used respectively is retained*/
+.first-level-header{
```
[the remark this answers is not in the document]
Comment (bl-aire) [reply binderhub/static/index.css:254]: Okay. I can change it.

I thought to use class selectors for specificity

File: binderhub/static/index.css (hunk 1 above)
Comment (bl-aire) [reply binderhub/static/index.css:250]: I will try to share. If I have any problem with it, I will ask for help.

Thanks for the review

**Comment by bl-aire** [comment]
> Wonderful, thank you! I've added some comments.
> 
> 
> 
> Let me know if you need a hand on the screenshot.

Thanks for the review. I'll implement changes mentioned

**Comment by minrk** [comment]
> I had issues with locating the links and donate button in the header. Please can you help me with that?

Yes! Those are not actually in this repo. They are added specifically for mybinder.org [here](https://github.com/jupyterhub/mybinder.org-deploy/blob/master/mybinder/values.yaml#L147-L165)

File: binderhub/static/index.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (minrk) [reply binderhub/static/index.css:254]: Class selectors are good if there is more than one kind of the same tag that should appear different. I don't think you need them if there's just the one tag for the one page.

[the remark this answers is not in the document]
Comment (bl-aire) [reply binderhub/static/index.css:254]: Okay.

I noticed another h1 in one of the html files. Will styling h1 also affect the tag below?

image

[the remark this answers is not in the document]
Comment (minrk) [reply binderhub/static/index.css:254]: It shouldn't because `index.css` is only applied to the main page, not error pages.

**Comment by bl-aire** [comment]
> > I had issues with locating the links and donate button in the header. Please can you help me with that?
> 
> 
> 
> Yes! Those are not actually in this repo. They are added specifically for mybinder.org [here](https://github.com/jupyterhub/mybinder.org-deploy/blob/master/mybinder/values.yaml#L147-L165)

Okay. I spent hours looking for it 😅

Thank you

File: binderhub/static/index.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (bl-aire) [reply binderhub/static/index.css:254]: Noted. I will make necessary changes to the code

**Comment by minrk** [comment]
Thank you, @bl-aire!

**Pull request merged**